src/app/demos/tabs.component.html

```diff
@@ -45,33 +45,32 @@ <h6>Vertical nav</h6>
 <hr>
 
 <h6>Tabs</h6>
-
-<ul class="nav nav-tabs" id="myTab" role="tablist">
-  <li class="nav-item" role="presentation">
-    <button class="nav-link active" id="home-tab" data-bs-toggle="tab" data-bs-target="#home" type="button" role="tab" aria-controls="home" aria-selected="true">Home</button>
+<ul ngbNav #nav="ngbNav" [(activeId)]="active" class="nav-tabs">
+  <li [ngbNavItem]="1">
+    <a ngbNavLink>Home</a>
+    <ng-template ngbNavContent>
+      <p>Raw denim you probably haven't heard of them jean shorts Austin. Nesciunt tofu stumptown aliqua, retro synth
```

Comment:
I would have had a `.mb-0` to avoid the extra bottom margin (3 tabs)

Reply:
I'll remove paragraph